tests/baselines/reference/declarationEmitReexportedSymlinkReference3.js

```diff
@@ -57,5 +57,8 @@ Object.defineProperty(exports, "__esModule", { value: true });
 __export(require("./keys"));
 
 
+//// [keys.d.ts]
+import { MetadataAccessor } from "@raymondfeng/pkg2";
+export declare const ADMIN: MetadataAccessor<boolean, import("../../pkg1/dist").IdType>;

```

Comment:
I am not sure if this is better or not.. 
cc: @weswigham @DanielRosenwasser @RyanCavanaugh

Reply:
It's probably not... I mean, technically it's correct, but when your code is seen in a structure like this (monorepo-y), you're not supposed to access sibling packages with a relative path, since the packages won't be siblings when shipped... The error was good in this case, since it called out that your types depended on a package you didn't directly depend on, which meant you needed to fix it in some way. Like this, it's.... Eck.

Reply:
Turns out we resolve that only because of how these tests run (all the sources are included irrespective of tsconfig file) when I add test using the tsc baseline I retain the error so should be ok? appending that test as a separate commit for reference 91c66a0

Reply:
Ping So this PR is good to go?

Reply:
I guess, yeah.